html/modules/custom/hpc_api/src/Query/EndpointPlanQuery.php

```diff
@@ -34,10 +34,17 @@ public function getPlanPublic($plan_id) {
       'end_date' => $data->planVersion->endDate ?? '',
       'orig_requirements' => $data->origRequirements ?? 0,
       'revised_requirements' => $data->revisedRequirements ?? 0,
+      'icon' => '',
       'categories' => [],
       'locations' => [],
     ];
 
+    // Add icon.
+    if (isset($data->governingEntities) && isset($data->governingEntities[0])) {
+      $plan['icon'] = $data->governingEntities[0]->governingEntityVersion->value->icon ?? '';
+    }
```

Comment:
See below, there are no plan icons, and fetching the first GVE icon to use as a plan icon is semantically wrong.